subt/drone.py

```diff
@@ -126,11 +136,18 @@ def update(self):
     def draw(self):
         import matplotlib.pyplot as plt
 
-        t = [a[0].total_seconds() for a in self.debug_arr]
-        height = [(a[1], a[1] - a[2], a[1] + a[3], a[4]) for a in self.debug_arr]
-        line_obj = plt.plot(t, height, '-o', linewidth=2)
-        plt.legend(iter(line_obj), ('altitude', 'alt - bottom', 'alt + top', 'pose3D.z'))
-        plt.xlabel('time (s)')
+        #t = [a[0].total_seconds() for a in self.debug_arr]
```

Comment:
we should introduce something like `--draw profile` or `--draw delay` ... i.e. optional `--draw` extra parameter (there are other modules where I am also commenting out graphs of different variables).

Reply:
Yes, good idea.

Reply:
linked to technical debt